app/med-list-create-component/med-list-create.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="med-list-create">
+    <button fab fab-bottom fab-right fab-fixed
+            secondary
+            class="med-list-create-fab"
+            [navPush]="onClickPage">
+        <ion-icon name="person-add"></ion-icon>
```

Comment:
I don't think there is a good med-specific add icon.

Just make this "add":

`<ion-icon name="add"></ion-icon>`